lessons/1.1-Variables/text.adoc

```diff
@@ -0,0 +1,15 @@
+= Variables
+
+Programs store data in variables. On line 5, a variable, called `the_answer` is
+_declared_ (using `int the_answer`) and _initialized_ with an _integer literal_
```

Comment:
Saying `int the_answer` *declares* is confusing. A declaration is a definition; a declaration can also come with an initializer `= 1`. Here, you make it sound like the two are distinct things. Given the amount of confusion surrounding declaration/definition esp. if they come from other languages, I would expand upon it here.